src/main/java/lingogo/model/flashcard/UniqueFlashcardList.java

```diff
@@ -129,12 +131,15 @@ public void downloadFlashcards(String fileName) throws CommandException {
      * Imports the contents of the CSV file in {@code filePath} to LingoGO!.
      * {@code filePath} must be a valid file path with .csv extension.
      */
-    public void uploadFlashcards(String filePath) throws CommandException {
+    public void importFlashcards(String filePath) throws CommandException {
         try {
             CSVReader reader = new CSVReaderBuilder(new FileReader(filePath)).withSkipLines(1).build();
```

Comment:
I think since you already created a `CommandException` for `MESSAGE_INVALID_CSV_FORMAT`, you can also check whether the first line matches `"Language, Foreign, English"` and then throw error CommandException when it doesnt match.